### DISCUSSION TITLE
Buttons on the Currency Calculator

### DISCUSSION THREAD

**Discussion by PranishKansakar** [PranishKansakar/Tafesa240154#8] [Ideas]
How many buttons do we need in the currency calculator? How does the buttons on the currency calculator work? Do they each do something specific to make it easier for people to convert currencies? I have planned to display 2 buttons: Currency Conversion and Exit.

**Comment by Omonotuaga** [comment]
You've got the right idea with two buttons! I think with just these two buttons, you're keeping things clean and easy for users to navigate. If you need any more help, just let me know.

Thank you! :)
